pandas/tests/indexes/base_class/test_constructors.py

```diff
@@ -7,14 +8,15 @@ class TestIndexConstructor:
     # Tests for the Index constructor, specifically for cases that do
     #  not return a subclass
 
-    def test_constructor_corner(self):
+    @pytest.mark.parametrize("value", [1, [1, 2][0], np.array([1, 2])[0]])
```

Comment:
```suggestion
    @pytest.mark.parametrize("value", [1, np.int64(1)])
```

I think can simplify a bit further